src/query/execute_statement.c

```diff
@@ -17133,3 +17074,95 @@ do_send_plan_trace_to_session (PARSER_CONTEXT * parser)
       free_and_init (plan_str);
     }
 }
+
+static int
```

Comment:
@lucianlazar2211 

Would you please add function comment for `do_insert_checks`? :)